src/resources/nfts/index.ts

```diff
@@ -20,8 +24,20 @@ const POLYGON_ALLOWLIST_STALE_TIME = 600000; // 10 minutes
 export const nftsQueryKey = ({ address }: { address: string }) =>
   createQueryKey('nfts', { address }, { persisterVersion: 1 });
 
+export const nftListingQueryKey = ({
+  contractAddress,
+  tokenId,
+  network,
+}: {
+  contractAddress: string;
+  tokenId: string;
+  network: Omit<Network, Network.goerli>;
+}) => createQueryKey('nftListing', { contractAddress, tokenId, network });
+
+export type PolygonAllowlist = { [key: string]: true };
```

Comment:
polygon allowlist structure got changed apparently

Reply:
If there are other edits to make in this PR, mind dumping this into a `@/resources/nfts/types` file? If no other edits, ignore for now to skip CI.